cypress/e2e/case-details/handle-triggers-and-exceptions/edit-fields-with-exceptions/edit-next-hearing-location.cy.ts

```diff
@@ -341,88 +332,65 @@ describe("NextHearingLocation", () => {
     cy.get("ul.moj-sub-navigation__list").contains("Offences").click()
     cy.get(".govuk-link").contains("Offence with HO100200 - Unrecognised Force or Station Code").click()
     cy.contains("td", "Next hearing location").siblings().should("include.text", "B@1EF$1")
-    cy.contains("td", "Next hearing location").siblings().contains("Editable Field").should("not.exist")
   })
 
-  context("Save correction button", () => {
-    it("Should be enabled when valid value is entered", () => {
-      loginAndVisit("/bichard/court-cases/0")
-
-      cy.get("ul.moj-sub-navigation__list").contains("Offences").click()
-      cy.get(".govuk-link").contains("Offence with HO100200 - Unrecognised Force or Station Code").click()
-      cy.get("#next-hearing-location").clear()
-      cy.get("#next-hearing-location").type("B43UY00")
-
-      cy.get("#save-next-hearing-location").should("be.enabled")
-    })
-
-    it("Should be disabled when no value is entered", () => {
-      loginAndVisit("/bichard/court-cases/0")
-
-      cy.get("ul.moj-sub-navigation__list").contains("Offences").click()
-      cy.get(".govuk-link").contains("Offence with HO100200 - Unrecognised Force or Station Code").click()
-      cy.get("#next-hearing-location").clear()
-      cy.get("#save-next-hearing-location").should("be.disabled")
+  it("Should display error message when Next Hearing Location is failed to auto-save", () => {
+    cy.intercept("PUT", "/bichard/api/court-cases/0/update", {
+      statusCode: 500
     })
 
-    it("Should be disabled when invalid value is entered", () => {
-      loginAndVisit("/bichard/court-cases/0")
-
-      cy.get("ul.moj-sub-navigation__list").contains("Offences").click()
-      cy.get(".govuk-link").contains("Offence with HO100200 - Unrecognised Force or Station Code").click()
-      cy.get("#next-hearing-location").clear()
-      cy.get("#next-hearing-location").type("B01")
-
-      cy.get("#save-next-hearing-location").should("be.disabled")
-    })
+    loginAndVisit("/bichard/court-cases/0")
 
-    it("Should be disabled when whitespace is entered", () => {
-      loginAndVisit("/bichard/court-cases/0")
+    cy.get("ul.moj-sub-navigation__list").contains("Offences").click()
+    cy.get(".govuk-link").contains("Offence with HO100200 - Unrecognised Force or Station Code").click()
+    cy.get("#next-hearing-location").clear()
+    cy.get("#next-hearing-location").type("B43UY00")
 
-      cy.get("ul.moj-sub-navigation__list").contains("Offences").click()
-      cy.get(".govuk-link").contains("Offence with HO100200 - Unrecognised Force or Station Code").click()
-      cy.get("#next-hearing-location").clear()
-      cy.get("#next-hearing-location").type("      ")
+    cy.get("#success-message").should("not.exist")
```

Comment:
Can there ever be two editable fields visible at once?

Reply:
Added ID on an editable field row to ensure correct block of error is being tested